dd-sdk-android-core/build.gradle.kts

```diff
@@ -170,9 +170,11 @@ dependencies {
     testImplementation(libs.bundles.testTools)
     unmock(libs.robolectric)
 
-    // Static Analysis
-    // TODO MTG-12 detekt(project(":tools:detekt"))
-    // TODO MTG-12 detekt(libs.detektCli)
+    // Test Fixtures
+    testFixturesImplementation(libs.kotlin)
+    testFixturesImplementation(libs.bundles.jUnit5)
+    testFixturesImplementation(libs.okHttp)
```

Comment:
it seems no class in `testFixtures` source set is using it, why do we need it?

Reply:
it's needed for something that's going to be added next but not in this PR yet